project/src/main/java/com/google/sps/servlets/CompetitionInfoServlet.java

```diff
@@ -14,25 +14,45 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
+import com.google.sps.data.*;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-import com.google.gson.Gson;
-import com.google.sps.data.*;  
 
 @WebServlet("/competitionInfo")
 public class CompetitionInfoServlet extends HttpServlet {
-
+  List<Competition> usersCompetitions = new ArrayList<>();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Competition compObj = new Competition(1895,"Google-Clouds-Comp","Emma Hogan","emmahogan@",0,0);
-
+    getUserCompetitions();
     Gson gson = new Gson();
-    
-    response.setContentType("text/html;");
 
-    response.getWriter().println(gson.toJson(compObj));
+    response.setContentType("application/json");
+
+    response.getWriter().println(gson.toJson(usersCompetitions));
+  }
+
+  private void getUserCompetitions() {
```

Comment:
Are you hardcoding some data to show in the UI?

Reply:
Yes I'm hardcoding data at the moment while waiting for the frontend.